ansible/playbook/tasks/update.yml

```diff
@@ -129,6 +129,16 @@
           - pmm-agent
         - ['stop', 'remove', 'add']
 
+    - name: check if fix-panels script exists
```

Comment:
It should not be needed: package is update above, and that file is always present now.
Or something weird is going on there.

Reply:
The play has been removed.

Reply:
Yepp, something else is going on there – tests fail. Do we switch repository to laboratory in tests? If not, we should.

Reply:
It has been switched to laboratory repository.